### DISCUSSION TITLE
issue: Focusing radio seems off

### DISCUSSION THREAD

**Discussion by Karolis92** [react-hook-form/react-hook-form#9561] [Q&A]
### Version Number

7.40.0

### Codesandbox/Expo snack

https://codesandbox.io/s/stoic-dew-h7wk6c

### Steps to reproduce

1. Go to codesandbox
2. Click on "focus" button
4. See focused radio
5. Try changing value using keyboard arrows.

### Expected behaviour

I would expect selected radio to be focused, but it's always first that is focused, so using keyboard to change value feels awkward

### What browsers are you seeing the problem on?

Firefox, Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bluebill1049** [comment]
There is no specific on focusing which radio input, if this is desired experience send us a feature request instead we can see how the community react to the request.

**Reply by Karolis92** [reply]
I would just expect it to be smooth experience for user
